pkg/tsdbctl/query.go

```diff
@@ -186,6 +179,10 @@ func (qc *queryCommandeer) newQuery(from, to, step int64) error {
 	if err != nil {
 		return errors.Wrap(err, "Failed to initialize the Querier object.")
 	}
+	aggregationWindow, err := utils.Str2duration(qc.aggregationWindow)
+	if err != nil {
+		return err
```

Comment:
We should probably say the issue was with the aggregation window in the error message (i.e. wrap the error).